stylus/components/wizard.styl

```diff
@@ -25,35 +25,28 @@ $wizard--waiting
 
 $wizard--scroll
     position absolute
+    padding-top rem(56)
 
     +medium-screen()
-        padding-top rem(80)
         box-sizing border-box
 
-    +tiny-screen()
+    @media (max-width: 33.9375rem), (max-height: 23.4375rem)
```

Comment:
What is the real target with this media query ? 

Since we're using it multiples times, do we need to create a new utility?

Reply:
Yep, sorry just a mixin. no need to add it to our generic collection.